EventFilter/EcalRawToDigi/plugins/MatacqProducer.cc

```diff
@@ -601,6 +602,8 @@ bool MatacqProducer::getMatacqFile(uint32_t runNumber, uint32_t orbitId, bool* f
         uint32_t firstOrb;
         uint32_t lastOrb;
         bool goodRange = getOrbitRange(firstOrb, lastOrb);
+        std::cout << "Get orbit range " << (goodRange ? "succeeded" : "failed") << ". Range: " << firstOrb << "..."
```

Comment:
please remind me where is this module running.
Is it just in private local test runs?

Note that most of other couts are placed behind `verbosity_` check.
Shouldn't this one be as well?

Reply:
It's running at P5 on a processing farm maintained by ECAL. (ECAL laser farm).

Reply:
concerning the cout. It is fine if no verbosity_ flag is checked. I will eventually add the check the day I want to remove  the message from the logs of the in-production running.